pkg/tests/cross-tests/rapid_tv_gen.go

```diff
@@ -389,6 +392,7 @@ func (tvg *tvGen) GenSchemaTransform() *rapid.Generator[schemaT] {
 		attrKind := tvg.GenAttrKind().Draw(t, "attrKind")
 		secret := rapid.Bool().Draw(t, "secret")
 		forceNew := rapid.Bool().Draw(t, "forceNew")
+		configModeAttr := rapid.Bool().Draw(t, "configMode")
```

Comment:
rapid has some finicky strange exceptions when you Draw a value and do not use the resulting entropy. I don't quite fully understand the mechanics yet. If it's not affecting this it's probably fine but if it surfaces it might be better to amend this to draw conditionally on tvg.generateConfigModeAttr and `s.Type == schema.TypeSet || s.Type == schema.TypeList` where the boolean drawn actually affects the program path.

Reply:
I haven't seen issues with this. I'll merge as is for now but keep this in mind. 🙏